### PR TITLE
community: Fix links in GraphVectorStore pydoc

### PR DESCRIPTION
**Pull request by cbornet**

### CODE REVIEW

File: libs/community/langchain_community/graph_vectorstores/__init__.py [hunk 1]
```diff
@@ -15,23 +18,21 @@
 
 Link extractors can be used to extract links from documents.
 
-Example:
-
-.. code-block:: python
+Example::
```
Comment (cbornet) [inline libs/community/langchain_community/graph_vectorstores/__init__.py:21]: Python is the default code-block language. So the syntax can be simplified.

File: docs/api_reference/scripts/custom_formatter.py [hunk 2]
```diff
@@ -17,15 +17,16 @@ def process_toc_h3_elements(html_content: str) -> str:
 
     # Process each element
     for element in toc_h3_elements:
-        element = element.a.code.span
-        # Get the text content of the element
-        content = element.get_text()
+        if element.a.code:
```
Comment (cbornet) [inline docs/api_reference/scripts/custom_formatter.py:20]: Pyoc with multiple levels of headings create `toc-h3` elements (even if there's no `.. contents` directive)

**Comment by cbornet** [comment]
Rebased.
Can this be reviewed 🙏 ?
Only easy to review docstring.

**Review by efriis — COMMENTED** [review]
I don't think the api reference formatter change does anything different aside from increasing the cases where the job could fail. Thoughts on reverting that and just doing the docstring change?

**Comment by cbornet** [comment]
I reverted the changes on the ref formatter.
Can you take a look again ?

**Pull request merged**